HopsanGUI/PlotTab.cpp

```diff
@@ -1449,10 +1449,10 @@ void PlotTab::openCreateBodePlotDialog()
     {
         const double dataSize = pTimeVector->getDataSize()+1;
         const double stopTime = pTimeVector->last();
-        const double maxFreq = dataSize/stopTime/2;
+        const double maxFreq = dataSize/stopTime/2*M_PI*2;
```

Comment:
Here is multiplied by 2PI but at line 5178 in hcomhandler the value was divided by 2PI. I have not looked at the code in more detail only this diff, so maybe its supposed to be that way?
OK I think I understand, see different comment.